development-guide.md

```diff
@@ -211,6 +211,13 @@ Where SVGs are used, they are used accessibly
 
 ### Images
 
+Decorative images should be implemented using the CSS `background-image` property, so that Assistive Technology can ignore the images. If a decorative image must be implemented using the HTML `<img>` element, you must ensure that:
```

Comment:
Hi @AshTNA - this looks good. I'd de-capitalise 'Assistive Technology' and change it to 'assistive technolog**ies**'

Reply:
Thanks @gtvj -  I have made this change.